testgres/node.py

```diff
@@ -413,6 +417,7 @@ def default_conf(self,
                      fsync=False,
                      unix_sockets=True,
                      allow_streaming=True,
+                     allow_logical=False,
```

Comment:
Why don't we enable this by default?

Reply:
Because it is not supported on postgres versions below 10 and there is specific message when someone's trying to enable this feature on those versions. Besides it produces extra WAL data and hence could work slightly slower.

Reply:
Ah, i see.